pom.xml

```diff
@@ -63,6 +63,11 @@
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+        </dependency>
```

Comment:
the dependency is needed only during runtime, could you specify this scope?

Reply:
fixed